linalg/densemat.hpp

```diff
@@ -275,6 +279,8 @@ class DenseMatrix : public Matrix
    void CopyMNDiag(double c, int n, int row_offset, int col_offset);
    /// Copy diag on the diagonal of size n to *this at row_offset, col_offset
    void CopyMNDiag(double *diag, int n, int row_offset, int col_offset);
+   /// Copy All rows and columns except m and n from A
+   void CopyExceptMN(const DenseMatrix &A, int m, int n);
```

Comment:
Actually, I do need it for something else.  I'm working on a side project involving General Relativity.  I use this method to extract 3x3 portions of a 4x4 tensor when building operators for a 3+1 splitting of the 4D operators.